portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted: " + textValue);
+
+    // Write the value to the response so the user can see it.
+    response.getWriter().println("You submitted: " + textValue);
+    response.sendRedirect("https://knguyen-sps-summer22.appspot.com/");
```

Comment:
nit: might useful to add some functionality so that when the user is redirected they get some kind of confirmation that the post succeeded (eg: "Your information was submitted" Somewhere on the redirect page)

Currently I don't think this `You submitted: ... ` text is actually returned to the user due to the redirect IIUC